core/src/main/java/com/amplifyframework/predictions/models/Landmark.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.models;
+
+import android.graphics.Point;
+import androidx.annotation.NonNull;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Class that holds the landmark detection results
+ * for the predictions category.
+ */
+public final class Landmark {
```

Comment:
The landmarks actually only have to do with faces. Can we call this `FacialFeatures`, or something? And have it contain `FacialFeature type;` and `List<Point> facialFeatureLocations`? 

Also, how do the points correspond to the type? Are the `List<Point>` actually a `Polygon`, describing the region of the picture that contains the facial feature?